charts/seed-bootstrap/aggregate-prometheus-rules/fluent-bit.rules.yaml

```
@@ -12,3 +12,44 @@ groups:
     annotations:
       description: "There are no fluent-bit pods running on seed: {{ .ExternalLabels.seed }}. No logs will be collected."
       summary: Fluent-bit is down
+  - alert: FluentBitReceivesLogsWithoutMetadata
+    expr: sum(increase(fluentbit_loki_gardener_logs_without_metadata_total[4m])) by (pod) > 0
```

To improve readability you can split up the expression. This isn't done for the other existing metrics, but we can start here :)
```suggestion
    expr: |
      sum by (pod) (
        increase(
          fluentbit_loki_gardener_logs_without_metadata_total[4m]
         )
       ) > 0
```